### PR TITLE
feat: APPS-2294 Add PageAnchor to default layout

### PR DESCRIPTION
**Pull request by jendiamond**
Connected to [APPS-2294](https://jira.library.ucla.edu/browse/APPS-2294)

1. pages/_.vue
1. https://deploy-preview-706--uclalibrary-test.netlify.app/about/policies/only-cats-allowed
   + pages/about/policies/_slug.vue
1. https://deploy-preview-706--uclalibrary-test.netlify.app/about/policies/
   + pages/about/policies/index.vue
1. pages/about/programs/_slug.vue
1. https://deploy-preview-706--uclalibrary-test.netlify.app/collections/explore/halloween-costumes
   + pages/collections/explore/_slug.vue
1. https://deploy-preview-706--uclalibrary-test.netlify.app/collections#section-title---fpb---associated-topics
    + pages/collections/index.vue
1. https://deploy-preview-706--uclalibrary-test.netlify.app/media-handling#
   + pages/help/_slug.vue
1. pages/help/services-resources/_slug.vue
1. https://deploy-preview-706--uclalibrary-test.netlify.app/visit/locations/film-television-archive
   + pages/visit/locations/_slug.vue

### CODE REVIEW

File: app/router.scrollBehavior.js
Comment (jendiamond) [inline app/router.scrollBehavior.js:1]: Deletes extra lines and white spaces

File: cypress/e2e/collectiondetailpage.cy.js [hunk 1]
```diff
@@ -31,4 +31,14 @@ describe("Collection Detail page", () => {
             cy.get("h1.error-title").should("contain","Page not found")
         })
     })
+
```
Comment (pghorpade) [inline cypress/e2e/collectiondetailpage.cy.js:34]: this does not belong to collection detail

File: nuxt.config.js [hunk 2]
```diff
@@ -66,6 +66,7 @@ export default {
         "~/plugins/craft.js",
         "~/plugins/data-api.js",
         "~/plugins/elasticsearchplugin.js",
+        { src: '~/plugins/get-headers.js', mode: 'client' },
```
Comment (pghorpade) [inline nuxt.config.js:69]: rename the file to be .client.js, and remove mode, this is pattern we are following

**Review by pghorpade — CHANGES_REQUESTED** [review]

File: cypress/e2e/collectionsexplorepage.cy.js [hunk 3]
```diff
@@ -4,7 +4,6 @@ describe("Explore Collection page", () => {
 
         // UCLA Library brand
         cy.get(".logo-ucla").should("be.visible")
-        cy.get(".page-collections-explore").should("be.visible")
```
Comment (pghorpade) [inline cypress/e2e/collectionsexplorepage.cy.js:7]: @jendiamond I am not sure why this is removed.

File: nuxt.config.js [hunk 4]
```diff
@@ -66,6 +66,7 @@ export default {
         "~/plugins/craft.js",
         "~/plugins/data-api.js",
         "~/plugins/elasticsearchplugin.js",
+        "~/plugins/get-headers.js",
```
Comment (pghorpade) [inline nuxt.config.js:69]: Rename it to get-headers.client.js

**Review by pghorpade — CHANGES_REQUESTED** [review]

**Review by pghorpade — APPROVED** [review]
💯 🍰

**Pull request merged**